test.html

```diff
@@ -26,7 +26,7 @@
   </p>
 
 
-  <form id="gform" method="POST" class="pure-form pure-form-stacked" action="https://script.google.com/macros/s/AKfycbyF7zsZXQUZLKBOaVKfsrzyr30FPHprgqd8flvC5WeuYQJnc5E/exec">
+  <form id="gform" method="POST" class="pure-form pure-form-stacked" data-email="contact.nelsonic+form.submit@gmail.com" action="https://script.google.com/macros/s/AKfycbyF7zsZXQUZLKBOaVKfsrzyr30FPHprgqd8flvC5WeuYQJnc5E/exec">
```

Comment:
Can do! I have a few more things to work on, and I will assign you once I have cleaned up and tested the code.